### ISSUE TITLE
Release Candidate Comments (New Contributors Closing Issues)

### ISSUE THREAD

**Issue by klumb** [chaoss/wg-evolution#292]
This issue was created to collect comments about the upcoming metrics release.

**This thread is for comments about New Contributors Closing Issues**

GitHub location: https://github.com/chaoss/wg-evolution/blob/master/metrics/New_Contributor_Closing_Issues.md

See all release candidates of metrics are at:
https://chaoss.community/metrics-202001/

**Comment by foundjem** [comment]
I have this minor concern other wise LGTM

During implementation should we consider both types of contributors “especially for contributors who are not also committers?”

Signed-off-by: Foundjem <foundjem@ieee.org>

**Comment by klumb** [comment]
@foundjem, what do you mean by "both types of contributors"?

**Comment by bproffitt** [comment]
Wondering if the title for this is correct. These are not new contributors, so much as contributors closing issues for the first time on a given project.

Also this metric feels incomplete. Are there no tool implementations?

**Comment by jgbarah** [comment]
I agree with you, @bproffitt. The idea was to keep a general name "New Contributors", and then a "category name" (Closing Issues), which would read as "new contributors in closing issues" or more clearly "persons closing issues for the first time". I don't know if it is better to keep all kinds of "new contributors" with a common name, or it is better to write it in the form which is easier to understand.

**Comment by klumb** [comment]
It would be nice to have common naming conventions for the metrics but I don't think we have been able to develop a usable structure so I would agree with @jgbarah and @bproffitt we should probably go with what is easiest to understand.

**Comment by klumb** [comment]
Is this metric ready for release?

**Comment by ccarterlandis** [comment]
@tommens has a PR still open for this metric (#297), do we want to wait just a little longer in case others have suggestions for the name, or go ahead and merge?

**Comment by GeorgLink** [comment]
PR was merged and is part of the release. Closing issue to get everything ready for release.

**Comment by ElizabethN** [comment]
This metric could use a refresh. It basically reads well, but there may be more context we want to give for getting this data. We don't mention Augur or GrimoireLab at all.
